dotcom-rendering/src/web/browser/dynamicImport/init.ts

```diff
@@ -10,36 +10,21 @@ const initialiseDynamicImport = () => {
 		window.guardianPolyfilledImport = new Function(
 			'url',
 			`return import(url)`,
-		) as (url: string) => Promise<any>;
+		) as (url: string) => Promise<unknown>;
```

Comment:
For my understanding, why are we doing this? Is it for safety?

Reply:
By typing as `unknown`, you have to do actual type checking to make the compiler happy.

Probably best to keep out for now, it's not directly related.

Reply:
An extra note here - you can read and write anything to `any`, as it's defined as being able to be anything. It is the most loose type you can get. 

`unknown` is the tightest type you can get, as you can't run most read / write operations on it. e.g:

```typescript
const wat: any = undefined
console.log(wat.things) // valid, but errors on runtime
console.log(wat + 0) // valid, but errors on runtime

const wot: unknown = undefined
console.log(wot.things) // invalid, can't compile
console.log(wot + 0) // invalid, can't compile
```

Reply:
Makes sense. Thanks both for your explanations!